.github/workflows/build-and-test.yml

```diff
@@ -70,21 +59,12 @@ jobs:
     name: Build and Test iPadOS
     uses: StanfordBDHG/.github/.github/workflows/xcodebuild-or-fastlane.yml@v2
     with:
+      runsonlabels: '["macOS", "self-hosted"]'
       path: 'Tests/UITests'
       scheme: TestApp
-      destination: 'platform=iOS Simulator,name=iPad Air 13-inch (M2)'
+      destination: 'platform=iOS Simulator,name=iPad Air 13-inch (M3)'
```

Comment:
A bit annoying that we always have to changes this but I don't see a nice way around this. But that might be adjustable in the template as well just to keep that up-to-date.